src/snippets/card-grid.njk

```diff
@@ -6,7 +6,7 @@ templateEngineOverride: md, njk
 
 For your use case, a series of `<article>` may be more appropriate. 
 
-If you are using a list as shown, and you want to retain list semantics for assistive technology users by having the current list item and it's order announced, adding `role="list"` is required. Otherwise, the use of `list-style: none` can remove list semantics for some assistive technology.
+If you are using a list as shown, and you want to retain list semantics for assistive technology for users by having the current list item and it's order announced, adding `role="list"` is required. Otherwise, the use of `list-style: none` can remove list semantics for some assistive technology.
```

Comment:
IMO as it was is more correct, or we could do:
```suggestion
If you are using a list as shown, and you want to retain list semantics for users of assistive technology by having the current list item and it's order announced, adding `role="list"` is required. Otherwise, the use of `list-style: none` can remove list semantics for some assistive technology.
```

Reply:
@eclectic-coding - hi! Do you have a preference for the choice above?